qt/app.py

```diff
@@ -735,12 +736,34 @@ def _create_menubar(self):
         restore.insertSeparator(self.act_restore_parent)
         restore.setToolTipsVisible(True)
 
+    def style_setter(self, point, toolbar):
+        """
+        Create a context menu for changing the style of the icons
+        """
+        context_menu = QMenu(self)
+        options = (
+            (_('Icon Only'), Qt.ToolButtonStyle.ToolButtonIconOnly),
+            (_('Text Only'), Qt.ToolButtonStyle.ToolButtonTextOnly),
+            (_('Text Under Icon'), Qt.ToolButtonStyle.ToolButtonTextUnderIcon),
+            (_('Text Beside Icon'), Qt.ToolButtonStyle.ToolButtonTextBesideIcon),
```

Comment:
I am not a native speaker but this sounds easier to me. What do you think?

```suggestion
            (_('Icons only'), Qt.ToolButtonStyle.ToolButtonIconOnly),
            (_('Text only'), Qt.ToolButtonStyle.ToolButtonTextOnly),
            (_('Text below icons'), Qt.ToolButtonStyle.ToolButtonTextUnderIcon),
            (_('Text beside icon'), Qt.ToolButtonStyle.ToolButtonTextBesideIcon),
```

Reply:
Yes, it reads much better